src/schemas/schema.json

```diff
@@ -1,262 +1,197 @@
 {
-  "title": "Resource Metadata",
```

Comment:
interesting that there are significant changes to the schema...

Reply:
This one is just a placeholder schema that I use to test the renderers, but indeed we'll have minor changes to our schemas.